frontend/apps/erd-web/.env.development

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_ENV_NAME=development
```

Comment:
The following is explained so that I've commit. Only secret is set from the Vercel dashboard.

> Good to know: .env, .env.development, and .env.production files should be included in your repository as they define defaults. 

ref: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#default-environment-variables